seml/config.py

```diff
@@ -262,12 +258,10 @@ def check_config(executable, conda_env, configs):
     # Extract experiment from module
     exps = [v for k, v in exp_module.__dict__.items() if type(v) == sacred.Experiment]
     if len(exps) == 0:
-        logging.error(f"Found no Sacred experiment. Something is wrong in '{executable}'.")
-        sys.exit(1)
+        raise ExecutableError(f"Found no Sacred experiment. Something is wrong in '{executable}'.")
     elif len(exps) > 1:
-        logging.error(f"Found more than 1 Sacred experiment in '{executable}'. "
-                      f"Can't check parameter configs. Disable via --no-config-check.")
-        sys.exit(1)
+        raise ExecutableError(f"Found more than 1 Sacred experiment in '{executable}'. "
+                              f"Can't check parameter configs. Disable via --no-config-check.")
```

Comment:
I think we've renamed `--no-config-check` to `--no-sanity-check`.

Reply:
Fixed in 2f09cb122a5edae451d7e3796c46d1f88dbeb0fe.